### ISSUE TITLE
Box Header Tools Link Color Problem

### ISSUE THREAD

**Issue by erayaydin** [almasaeed2010/AdminLTE#223]
> Sorry if that issue already open

Hello,
In `box` section

```
<div class="box-tools pull-right">
    <a class="btn btn-primary btn-xs"><i class="fa fa-plus"></i> Create</a>
</div>
```

`Create' color is 'black'. Because of this, can not read text.

![Screenshot](http://i.imgur.com/fj3aEjY.png)

**Comment by thePanz** [comment]
I can confirm the issue. It was introduced in latest releases (previous of 1.3.x)
Solved by re-introducing this CSS fragment (found in a previous release of the theme):

```css
.box .box-header a:not(.btn) {
  color: #444;
}
```

in AdminLTE.css, around line 1444

**Comment by erayaydin** [comment]
Same way for  solve this issue. But I think it should be add in AdminLTE repo.

**Comment by almasaeed2010** [comment]
This issue has been resolved in a future release (2.0).

**Comment by erayaydin** [comment]
Thank you @almasaeed2010 :+1:
